awx/settings/defaults.py

```diff
@@ -422,6 +422,7 @@ def IS_TESTING(argv=None):
 # heartbeat period can factor into some forms of logic, so it is maintained as a setting here
 CLUSTER_NODE_HEARTBEAT_PERIOD = 60
 RECEPTOR_SERVICE_ADVERTISEMENT_PERIOD = 60  # https://github.com/ansible/receptor/blob/aa1d589e154d8a0cb99a220aff8f98faf2273be6/pkg/netceptor/netceptor.go#L34
+EXECUTION_NODE_REMEDIATION_CHECKS = 60 * 10  # once every 10 minutes check if an execution node errors have been resolved
```

Comment:
Is this something we should render in the UI Settings?

Reply:
Out of these 3 settings, this is the only one that would be valid to implement as a CTiT setting. It will not require a restart in order to take effect.

I don't want to volunteer to do this, because I don't know if it's worth it. But yes, we could put this in UI settings.